connectors/salesforce/src/main/resources/cards/text.properties

```diff
@@ -17,5 +17,12 @@ opportunity.closedate=Close Date
 opportunity.stage=Sales stage
 opportunity.amount=Amount
 opportunity.comments=Feed Comments
+opportunity.expected.revenue=Expected Revenue
+
+opportunity.update.nextstep=Update Next Steps
+opportunity.update.nextstep.label=Update Next Steps
+
+opportunity.update.closedate=Update Close Date
+opportunity.update.closedate.label=Update Close Date
```

Comment:
`opportunity.update.closedate.label` not used ?
Same with nextstep

Reply:
```
.setLabel(this.cardTextAccessor.getActionLabel("opportunity.update.closedate", locale))
```

getActionLabel adds `+ ".label"` to the message key.